app/_hub/kong-inc/jq/index.md

```diff
@@ -0,0 +1,101 @@
+---
+name: Kong jq
```

Comment:
```suggestion
name: jq
```
I'd avoid putting "Kong" in the plugin name, we don't normally do that for plugins.

Reply:
Ok, my thinking was that having a plugin named two lowercase letters is odd - @eskibars any preference for the name of this plugin?

Reply:
k, will update